docs/CSBR.md

```diff
@@ -503,19 +503,16 @@ If the CA decides that the revocation will have an unreasonable impact on its cu
 
 The CA MUST revoke a Code Signing Certificate within one (1) business day if the Subscriber requests in writing that the CA revoke the Certificate or notifies the CA that the original certificate request was not authorized and does not retroactively grant authorization.
 
-#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Malware
+#### 4.9.1.3 Revocation Based on Reported or Detected Compromise or Use in Suspect Code
 
-For all incidents involving malware, CAs SHALL revoke the Code Signing Certificate in accordance with and within the following maximum timeframes. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these timeframes.
+Except for cases that fall under Section 4.9.1.1, all incidents that lead the CA to believe that the certificate private key is compromised or is being used for Suspect Code, CAs SHALL revoke the Code Signing Certificate in accordance with and within the following maximum timeframes. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these timeframes.

```

Comment:
```suggestion
Except for cases that fall under Section 4.9.1.1, all incidents that lead the CA to believe that the Subscriber's Private Key is compromised or is being used for Suspect Code, CAs SHALL revoke the corresponding Code Signing Certificate in accordance with and within the following maximum time frames. Nothing herein prohibits a CA from revoking a Code Signing Certificate prior to these time frames.

```

Reply:
Good catch. Thanks!

Reply:
Resolved in [8bd87c3](https://github.com/cabforum/code-signing/pull/10/commits/8bd87c3e3a76236d79aecdc34362b8b2fc8b7daf)